sdk/search/Azure.Search.Documents/tests/Samples/Sample01_HelloWorld.cs

```diff
@@ -466,5 +466,62 @@ public async Task GetCountAsync()
             Console.WriteLine($"Search index {indexName} has {count.Value} documents.");
             #endregion Snippet:Azure_Search_Tests_Samples_GetCountAsync
         }
+
+        [Test]
+        public async Task QuerySession()
+        {
+            const int size = 150;
+            await using SearchResources resources = await SearchResources.CreateLargeHotelsIndexAsync(this, size, false, true);
+            Environment.SetEnvironmentVariable("SEARCH_ENDPOINT", resources.Endpoint.ToString());
+            Environment.SetEnvironmentVariable("SEARCH_API_KEY", resources.PrimaryApiKey);
+            Environment.SetEnvironmentVariable("SEARCH_INDEX", resources.IndexName);
+
+            HashSet<string> uniqueDocuments = new HashSet<string>();
+            bool hasDuplicates = false;
+
+            #region Snippet:Azure_Search_Tests_Samples_QuerySession
+            Uri endpoint = new Uri(Environment.GetEnvironmentVariable("SEARCH_ENDPOINT"));
+            AzureKeyCredential credential = new AzureKeyCredential(Environment.GetEnvironmentVariable("SEARCH_API_KEY"));
+            string indexName = Environment.GetEnvironmentVariable("SEARCH_INDEX");
+
+            SearchClient searchClient = new SearchClient(endpoint, indexName, credential);
+#if !SNIPPET
+            searchClient = InstrumentClient(new SearchClient(endpoint, indexName, credential, GetSearchClientOptions()));
+#endif
+            SearchResults<Hotel> results = await searchClient.SearchAsync<Hotel>(
```

Comment:
@tg-msft @KrzysztofCwalina are we switching to async samples now? I've always thought we should, but to my knowledge we've always opted first and foremost for sync for "ease".

Reply:
Most of samples are async in Search:
* [Index Operations](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/search/Azure.Search.Documents/samples/Sample03_Index.md)
* [Indexing Documents](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/search/Azure.Search.Documents/samples/Sample05_IndexingDocuments.md)
* [All the Vector Search samples](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/search/Azure.Search.Documents/samples/Sample07_VectorSearch.md)
* [Semantic Search](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/search/Azure.Search.Documents/samples/Sample08_SemanticSearch.md)